Tools/TestWebKitAPI/Tests/WebKitCocoa/FindInPageAPI.mm

```diff
@@ -268,6 +269,20 @@
 #if ENABLE(IMAGE_ANALYSIS)
 static unsigned gDidProcessRequestCount = 0;
 
+@interface FindDelegate : NSObject<_WKFindDelegate> {
+    @public int _numCallsToDidFindMatches;
```

Comment:
Though this is just testing code, it would be nice to clean this up using (more modern) ObjC idioms:

```
@interface FindDelegate : NSObject<_WKFindDelegate>
@property (nonatomic) int numberOfCallsToDidFindMatches;
@end

@implementation FindDelegate

 - (void)_webView:(WKWebView *)webView didFindMatches:(NSUInteger)matches forString:(NSString *)string withMatchIndex:(NSInteger)matchIndex
 {
     _numCallsToDidFindMatches++;
 }

 @end
```

(…and later…)

```
        return (gDidProcessRequestCount == 1 && [findDelegate numberOfCallsToDidFindMatches] == 2);
     }, 1, @"Timed out waiting for image analysis to start and for search for string to be performed twice.");
```